automated_testing/test/pageobjects/auth-page.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable class-methods-use-this */
+import { ChainablePromiseElement } from 'webdriverio';
+import selectors from '../constants/auth-page-selectors';
+import Page from './page';
+
+class AuthPage extends Page {
+    private pageLocation = '/Account/Login';
```

Comment:
React router uses lowercase ( /account/login ). I suggest this is changes because we have routes for the backend spelled in title case and we might confuse them.